langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockChatModelIT.java

```diff
@@ -77,6 +88,99 @@ void testBedrockAnthropicV3SonnetChatModelImageContent() {
         assertThat(response.finishReason()).isIn(FinishReason.STOP, FinishReason.LENGTH);
     }
 
+    @Test
+    void testFunctionCallingWithBedrockAnthropicV3SonnetChatModel() {
```

Comment:
please add a test where tool does not have parameters:

```java
ToolSpecification clock = ToolSpecification.builder()
                .name("current_time")
                .description("returns current time")
                .build();
```

Reply:
You can simply add `BedrockAnthropicMessageChatModel` to `AiServicesWithToolsIT.models()`, run all tests and see if it works as expected

Reply:
> please add a test where tool does not have parameters:
> 
> ```java
> ToolSpecification clock = ToolSpecification.builder()
>                 .name("current_time")
>                 .description("returns current time")
>                 .build();
> ```

Done.

Reply:
> You can simply add `BedrockAnthropicMessageChatModel` to `AiServicesWithToolsIT.models()`, run all tests and see if it works as expected

In development. Anthropic Claude 3 does not support string arrays.

Reply:
> You can simply add `BedrockAnthropicMessageChatModel` to `AiServicesWithToolsIT.models()`, run all tests and see if it works as expected

Implemented.
![2024-09-15_05-43-05](https://github.com/user-attachments/assets/17e9c384-a7a8-42f2-9d2b-4b1d24b73170)